services/ext/api.go

```diff
@@ -1546,6 +1546,35 @@ func (api *PublicAPI) CreateTokenGatedCommunity() (*protocol.MessengerResponse,
 	return api.service.messenger.CreateTokenGatedCommunity()
 }
 
+// Returns response with AC notification when owner token is received
+func (api *PublicAPI) RegisterOwnerTokenReceivedNotification(communityID string) (*protocol.MessengerResponse, error) {
+	return api.service.messenger.RegisterOwnerTokenReceivedNotification(communityID)
+}
+
+// Returns response with AC notification when setting signer is successfull
+func (api *PublicAPI) RegisterSetSignerSuccessNotification(communityID string) (*protocol.MessengerResponse, error) {
+	return api.service.messenger.RegisterSetSignerSuccessNotification(communityID)
+}
+
+// Returns response with AC notification when setting signer is failed
+func (api *PublicAPI) RegisterSetSignerFailedNotification(communityID string) (*protocol.MessengerResponse, error) {
+	return api.service.messenger.RegisterSetSignerFailedNotification(communityID)
+}
+
+// Returns response with AC notification when setting signer is declined
+func (api *PublicAPI) RegisterSetSignerDeclinedNotification(communityID string) (*protocol.MessengerResponse, error) {
+	return api.service.messenger.RegisterSetSignerDeclinedNotification(communityID)
+}
```

Comment:
These shouldn't be needed as API, it should be determined behind the scenes. See: https://github.com/status-im/status-go/pull/4187#discussion_r1370598171